packages/main/src/Token.js

```diff
@@ -114,6 +135,14 @@ class Token extends UI5Element {
 		this.i18nBundle = getI18nBundle("@ui5/webcomponents");
 	}
 
+	_handleSelect() {
```

Comment:
I thin we should fire an event both cases - when the user selects and unselects a token. So, _handleEvent and _select can be combined in one:

```js
	this.selected = !this.selected;
	this.fireEvent("select");
```

I saw that we have such event name in the RadioButton already. In the CheckBox we have "change", let's keep it "select" for now.